pyproject.toml

```diff
@@ -14,7 +14,7 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7.2"
+python = "^3.9.0"
```

Comment:
I can bring this down to 3.8.0 if we think it matters.

Reply:
Not sure if this means we have to up the python prereq from 3.7.2, as stated in the doc:

> Python version: >= 3.7.2